roles/acl/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+  - name: install acl
```

Comment:
Can you also specify the OS version in the task name? e.g. https://github.com/osu-mist/ansible-roles/blob/master/roles/openjdk-8/tasks/main.yml#L2